docs/dashboards/scheduled-report.md

```diff
@@ -20,45 +20,74 @@ If a .pdf and/or .png export fails, an error email will be sent to your email ID
+   * **Select a frequency**. Choose the frequency settings for the scheduled dashboard report: 
+      * **Frequency**. Choose the frequency Daily, Weekly, or Monthly.
+      * **Every.** The options depend on the Frequency you selected.
+         * **Daily.** Choose Every day, Weekdays, or Weekends.
+         * **Weekly.** Choose a particular day of the week.
+         * **Monthly.** Choose which day of which week of the month you want the report to be sent.
+      * **Time.** Choose the time of day you want the dashboard to be sent.
+      * **Time Zone.** Choose your time zone.
+   * **Add recipients**. 
+      * **Email to.** Enter the email addresses where you want the dashboard to be sent.
+      * **Subject.** Enter a subject for the email.
+      * **Message.** Enter the body of the email.
+1. Click **Schedule** to schedule the report.
+1. (Optional) Click **Schedule New Report** to create multiple scheduled reports for the dashboard. By allowing multiple reports from a single dashboard, you can save time, reduce effort, and ensure the delivery of critical insights to your team members and stakeholders. With scheduled reports, you can keep everyone informed with daily updates for your operations team or monthly summaries for executive reviews.<br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/scheduled_report_1.png')} style={{border: '1px solid gray'}} alt="scheduled_report_1>" width="600"/>
 
 :::note
 Scheduled reports are allowed to run for a maximum of five minutes. Any panel that has not completed running its queries at this time will have partial or empty data in the panel with an error message in the report.
 :::
 
 ### Limitations
 
-* You can schedule only one export per dashboard.
-* Altering the queries or template variables will not update the scheduled report.
-* You can enter up to 20 email addresses to which you want to send the report.
+* Altering the template variables in dashboard will not update the scheduled report.
+* You can enter up to 10 email addresses to which you want to send the dashboard.
+* You can create a maximum of 20 scheduled reports for a dashboard.
 
 ## Edit a scheduled report
 
 To update an existing a scheduled report dashboard email:
 
-1. Navigate to the dashboard, then click the kebab menu and select **Update Scheduled Report**.
-1. Make desired changes and click **Update** to save them. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report.png')} alt="Update button on the Schedule Report dialog" width="450"/>
+1. Navigate to the dashboard, then click the kebab menu and select **Edit Reports**.
+1. Click the kebab menu and select **Edit** against the required report. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report_1.png')} alt="update_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
```

Comment:
Not sure if it's worth mentioning here, but the user can also double click the row to edit report.

Reply:
Added both the functions to edit the report.